db/migrate/20250403053114_add_index_to_messages.rb

```diff
@@ -0,0 +1,13 @@
+class AddIndexToMessages < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
```

Comment:
I don't think this statement has any effect here, the worker will run outside whatever connection this migration makes